test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxserviceTests.cs

```diff
@@ -253,10 +253,16 @@ private async Task<Mock<RequestContext<DacFxResult>>> SendAndValidateExtractInva
 
             DacFxService service = new DacFxService();
             ExtractOperation operation = new ExtractOperation(extractParams, result.ConnectionInfo);
```

Comment:
Do you have to have an integration test to validate a version? Could the operation be tested without the DacFxService?

Reply:
A test to validate a valid version?

Reply:
I removed the integration test and added a unit test to validate the version parsing.